dataflow/src/ops/rewrite.rs

```diff
@@ -0,0 +1,240 @@
+use std::collections::{HashMap, HashSet};
+use prelude::*;
+
+#[derive(Debug, Clone, Serialize, Deserialize)]
+pub struct Rewrite {
+    src: IndexPair,
+    should_rewrite: IndexPair,
+
+    rw_col: usize,
+    value: DataType,
+
+    key: usize,
+}
+
+impl Rewrite {
+    pub fn new(src: NodeIndex, should_rewrite: NodeIndex, rw_col: usize, value: DataType, key: usize) -> Rewrite {
+        Rewrite {
+            src: src.into(),
+            should_rewrite: should_rewrite.into(),
+            rw_col: rw_col,
+            value: value,
+            key: key,
+        }
+    }
+
+    fn rewrite(&self, r: Vec<DataType>) -> Vec<DataType> {
+        let mut r = r.clone();
+        r[self.rw_col] = self.value.clone();
+        r
+    }
+
+    fn generate_row(&self, r: Vec<DataType>, positive: bool) -> Record {
```

Comment:
I believe `(Vec<DataType>, bool)` already implements `Into<Record>`, so wherever you call this method, you should be able to just use `.into()` instead :)